lib/_cookieUtils.ts

```diff
@@ -49,6 +50,12 @@ export function setUserToken(userToken: string | number): void {
   }
 }
 
-export function getUserToken() {
+export function getUserToken(
```

Comment:
Why theses changes to add a callback to `getUserToken`? I have the feeling that it could have been another PR :thinking:

Reply:
The purpose of the PR is to remove the callback from the queue part and move it on the method level. The only method using the callback right now is `getUserToken`. 

That why I called it, `fix(processQueue): move callback on method level` which is probably a terrible name :)

I can split it in two PR but it breaks the feature continuity insured by this test: https://github.com/algolia/search-insights.js/pull/114/files

Reply:
Ok if this break a feature then it makes sense to keep both in the same PR :+1: